kvbc/src/reconfiguration_kvbc_handler.cpp

```diff
@@ -567,8 +567,10 @@ bool ReconfigurationHandler::handle(const concord::messages::ClientKeyExchangeSt
   for (const auto& gr : bftEngine::ReplicaConfig::instance().clientGroups) {
     for (auto cid : gr.second) {
       if (command.tls) {
-        std::string client_cert_path = bftEngine::ReplicaConfig::instance().certificatesRootPath + "/" +
-                                       std::to_string(cid) + "/client/client.cert";
+        const std::string base_path = bftEngine::ReplicaConfig::instance().certificatesRootPath;
+        std::string client_cert_path = (bftEngine::ReplicaConfig::instance().useUnifiedCertificates)
+                                           ? base_path + "/" + std::to_string(cid) + "/node.cert"
+                                           : base_path + "/" + std::to_string(cid) + "/client/client.cert";
         auto cert = psm.decryptFile(client_cert_path).value_or("invalid client id");
```

Comment:
Please include "/" + std::to_string(cid) in base_path for a simplification, as recommended above.